fsfuzz

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
@stevegrubb bash environment works in ubuntu 16.04 can you please adopt this change .